### PR TITLE
feature: password show/hide toggle

### PR DESCRIPTION
**Pull request by jerichocalibuso**
Hi Natalie, I've implemented show/hide toggle in both password field only in SignUp page since SignIn is still under construction. Although it is working, I think my code needs to be refactored but I have no idea what is the best practice for the code. I appreciate any suggestion. Thanks!

### CODE REVIEW

File: src/screens/SignUp/index.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
   const [user, setUser] = useState({
-    name: '',
-    email: '',
-    password: '',
-    passwordConfirm: '',
+    name: "",
+    email: "",
+    password: "",
+    passwordConfirm: "",
+    isPasswordShown: false,
```
Comment (vyasriday) [inline src/screens/SignUp/index.js:11]: A suggestion. You can name variables like `showPassword` and `showConfirmPassword`

File: src/screens/SignUp/index.js [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 const Signup = () => {
   const [user, setUser] = useState({
-    name: '',
-    email: '',
-    password: '',
-    passwordConfirm: '',
+    name: "",
+    email: "",
+    password: "",
+    passwordConfirm: "",
```
Comment (vyasriday) [inline src/screens/SignUp/index.js:10]: I think `confirmPassword` is better name

File: src/screens/SignUp/index.js (hunk 1 above)
Comment (jerichocalibuso) [reply src/screens/SignUp/index.js]: Updated, thanks!

File: src/screens/SignUp/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jerichocalibuso) [reply src/screens/SignUp/index.js:10]: Also updated, thank you.

File: src/screens/SignUp/index.js [hunk 3]
```diff
@@ -20,6 +29,79 @@ const Signup = () => {
...
+      >
+        <path d="M0 0h24v24H0z" fill="none" />
+        <path d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5zM12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z" />
+      </svg>
+    ),
+    toHidePassword: (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 24 24"
+        fill="black"
```
Comment (nataliepina) [inline src/screens/SignUp/index.js:59]: Let's change the fill on the icons. The black demands a bit too much attention from the eye first thing. Make it a lighter shade of gray ideally.

Comment (jerichocalibuso) [reply src/screens/SignUp/index.js]: Changed it from black to gray. What do you think?

Comment (nataliepina) [reply src/screens/SignUp/index.js]: I went in and added a fill on the class on the svg. gray is good.

**Pull request merged**